client/components/UserPage.js

```diff
@@ -37,17 +38,38 @@ const _UserPage = ({ user, userChallenges, fetchUserChallenges }) => {
           </button>
         )}
       </div>
-      <ul>
-        {userChallenges.map(uc => (
-          <li>{uc.name}</li>
-        ))}
-      </ul>
+      <div className="d-flex flex-column align-items-center">
+        <h1>My Completed Challenges</h1>
+        <table className="table">
```

Comment:
@chem2016 please refactor this into a separate component (UserSolutionTable or something along those lines). Once you're done with that I think this is good to go